cognite/client/data_classes/workflows.py

```diff
@@ -560,6 +605,49 @@ def dump(self, camel_case: bool = True) -> dict[str, Any]:
         }
 
 
+class SimulationTaskOutput(WorkflowTaskOutput):
+    """
+    The class represent the output of Simulation execution.
+    Args:
+        run_id (int | None): The run id of the simulation execution in the SimInt API.
+        logs (list[dict[str, Any]] | None): Logs from the simulation execution.
+        status_message (str | None): Status of the current simulation execution.
+        outputs (list[dict[str, Any]] | None): Outputs results from the simulation execution
+    """
+
+    task_type: ClassVar[str] = "simulator"
+
+    def __init__(
+        self,
+        run_id: int | None,
+        logs: list[dict[str, Any]] | None,
+        status_message: str | None,
+        outputs: list[dict[str, Any]] | None,
```

Comment:
Both of these are not arbitrary objects. You need to add SimulationLog and SimulationRunResultData
![image](https://github.com/user-attachments/assets/b43c9f5e-74fe-48ad-a40f-60dd96400408)